src/main.ts

```diff
@@ -85,6 +86,9 @@ export class Purchases {
   /** @internal */
   private _appUserId: string;
 
+  /** @internal */
+  private _brandingInfo: BrandingInfoResponse | null = null;
```

Comment:
This is now cached in memory, but I believe it should be ok, since this shouldn't change right now (except maybe with a different API key)